### DISCUSSION TITLE
Different modes for USB C charging?

### DISCUSSION THREAD

**Discussion by ggjiza** [seerge/g-helper#901] [Ideas]
I'm not pretty sure, but I believe AC used to set different power mode to USB C charging than DC.
My M16 would set balanced when using type C, Turbo on DC.
Having the option to link/unlink the two would be useful as mine discharge on type C/turbo combo.

**Comment by MrxSiN** [comment]
Bump this. Since there a GPU clock limit in GHelper, having different mode for type C charging is huge plus. 
Since we can limit GPU Freq and CPU power limit, with proper setup, gaming while on type c charger will charge the laptop instead of  discharging due to heavy power draw.

**Comment by gnartS** [comment]
Absolutely.

My G16 2023 supports USB-C charging and power - above 100W it takes from the battery and tops back up (not good) but below 100W and it doesnt.  A profile to set for USB-C (or custom profiles) where i can limit power would be great.

The laptop does recognise USB-C as it flashes up "PD Charging" with AC when you use it.

**Reply by cosmicAP** [reply]
my asus tuf dash f15 also supports 100 watt charging by type-c , but i don't know if it supports by-pass or not . how did you know it don't take juice from the battery under load?

**Reply by gnartS** [reply]
If i pull over 100W i can see occasionally the orange charge light come on.  Under that it just stays on the normal white as it does when fully charged.

**Reply by cosmicAP** [reply]
ahh got it , i just hope asus tell us if it supports by-pass or not .

**Reply by gnartS** [reply]
It would help but anecdotally from my use i dont see it charging unless i start pulling power.
At the very least a PD profile would help tune so its not discharging/recharging anywhere  near as much.

Currently when on PD i have to remember to  manually alter one of the existing profiles and remember to put it back later.
